### ISSUE TITLE
Fix `libsecp256k1` error when installing project

### ISSUE THREAD

**Issue by germsvel** [mana-ethereum/mana#12]
When initially compiling the mana project, `libsecp256k1` fails to compile. 

The current workaround is to compile it manually (as is done in [exthereum's blockchain](https://github.com/exthereum/blockchain#installation)), 

```
export "CFLAGS=-I/usr/local/include -L/usr/local/lib"
cd deps/libsecp256k1 && rebar compile
mix compile
```

but we should find a more permanent solution. 

One possible solution suggested by @ayrat555 is to set the manager in the dependency, 

```
{:libsecp256k1, [github: "exthereum/libsecp256k1", manager: :rebar]}
```

But I know this has the potential problems as well, if any of the other dependencies are using `libsecp256k1` and it is getting it in a different way.

**Comment by germsvel** [comment]
@ayrat555 does https://github.com/poanetwork/mana/pull/371 fix this?

**Comment by hayesgm** [comment]
This should be fixed with 0.1.10 version of libsecp256k1.
